website/docs/d/standards_resources_scores.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_standards_resources_scores"
+sidebar_current: "docs-pagerduty-datasource-standards-resources-scores"
+description: |-
+  Get information about the scores received for the standards associated to
+  multiple resources.
+---
+
+# pagerduty\_standards\_resource\_scores
+
+Use this data source to get information about the [scores for the standards for
+many resources][1].
+
+## Example Usage
+
+```hcl
+data "pagerduty_standards_resources_scores" "scores" {
+    resource_type = "technical_services"
+    ids = ["P703E9Q", "PR6MHNF", "PQVUB8D"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `resource_type` - Type of the object the standards are associated to. Allowed values are `technical_service`.
+* `ids` - List of identifiers of the resources to query.
+
+## Attributes Reference
+
+* `resources` - List of score results for each queried resource.
+
+### Resource (`resources`) is a map with following attributes:
+
+* `resource_type` - Type of the object the standards are associated to. Allowed values are `technical_service`.
```

Comment:
Please mind the typo in `technical_service`, shouldn't it be `technical_services`?